libraries/splunk_app.rb

```diff
@@ -325,7 +325,7 @@ def hash_to_proc(source_module, data, context = {})
         proc_sym = data['proc'].to_sym
         data = symbolize_keys(data).reject { |k, _| k == :proc }
         arguments = context.merge data
-        source_module.send proc_sym, arguments
+        source_module.send proc_sym, **arguments
```

Comment:
While doing a bit more testing, I noticed this warning showing up in the chef output under Chef 16/Ruby2.7.1:
```
/var/chef/cache/cookbooks/cerner_splunk/libraries/splunk_app.rb:328: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```
This change resolves that warning.